packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -55,11 +55,14 @@ import { RestrictedByPermissions } from '@commercetools-frontend/permissions';
 import { location } from '../../utils/location';
 import { GtmContext } from '../gtm-booter';
 import LoadingPlaceholder from '../loading-placeholder';
-import styles from './navbar.mod.css';
 import messages from './messages';
 import useLoadingMenuLayoutEffect from './use-loading-menu-layout-effect';
 import useNavbarStateManager from './use-navbar-state-manager';
 import nonNullable from './non-nullable';
+// https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros
+import compiledStyles from /* preval */ './navbar.styles';
+
+const styles = compiledStyles.jsonMap;
```

Comment:
In the code, we reference the class names using the JSON map.

Bonus now is that we also get autocompletion and type checks.